storm/src/main/storm/mesos/MesosCommon.java

```diff
@@ -72,8 +72,15 @@ public static String getMesosComponentNameDelimiter(Map conf, TopologyDetails in
         .or(DEFAULT_MESOS_COMPONENT_NAME_DELIMITER);
   }
 
+  public static String timestampMillis() {
```

Comment:
* slf4j-api: https://github.com/mesos/storm/issues/98
* String.format: https://github.com/mesos/storm/issues/99

Also, thanks for the recommendation on TimeUnit, I've switched to this instead of the math-ey logic:

```
    long now = System.currentTimeMillis();
    long secs = TimeUnit.MILLISECONDS.toSeconds(now);
    long msecs = (TimeUnit.MILLISECONDS.toMillis(now) -
                            TimeUnit.SECONDS.toMillis(TimeUnit.MILLISECONDS.toSeconds(now)));
    return String.format("%d.%03d", secs, msecs);
```